modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/Diagram.js

```diff
@@ -75,10 +77,56 @@ const Diagram = ({
 		radius: newPinSettings.defaultRadius,
 	});
 
+	const importPinSchema = (svg) => {
+		const textDatas = [];
+		const pinDatas = [];
+		const parser = new DOMParser();
+
+		const xmlImage = parser.parseFromString(svg, 'image/svg+xml');
+
+		const rootLevel = xmlImage.getElementById('Livello_Testi');
```

Comment:
This `id` seems really specific and I am afraid this will cause regressions. What is this value coming from?

Reply:
The id is coming from an SVG file from CIFA. We implemented this solution only for this customer, I agree that this may cause some regression in the future, but we have on the roadmap to move that id to an external property.